src/core/Debug.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  Disposable,
+  ExtensionMode,
+  Location,
+  TextEditorSelectionChangeEvent,
+  window,
+  workspace,
+} from "vscode";
+import { SyntaxNode, TreeCursor } from "web-tree-sitter";
+import { Graph } from "../typings/Types";
+
+const originalDebugLog = console.debug;
+const disabledDebugLog = () => {};
+
+console.debug = disabledDebugLog;
```

Comment:
Why do you set this here?

Reply:
To remove debugs from things that are initialized before this. The tree sitter for example

Reply:
Hmm. But what if you actually want to see those things?

Reply:
What if you don't? I have now removed at line